### PR TITLE
DAOS-3656 test: enable HDF5 vol unit tests

### PR DESCRIPTION
**Pull request by mjean308**
PR-repos: hdf5-vol-daos@PR-1
Test-tag-hw-small: vol
Skip-checkpatch: true
Skip-run_test: true
Skip-func-test: true
Skip-func-hw-test-medium: true
Skip-func-hw-test-large: true

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

### CODE REVIEW

**Comment by mjean308** [comment]
HDF5 unit tests have passed successfully.
The two failures are known issues: 

ior small :
DAOS-5429

daos core rebuild server crash
DAOS-5452

File: src/tests/ftest/util/vol_test_base.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        # VOL needs to run from a file system that supports xattr.
+        #  Currently nfs does not have this attribute so it was recommended
+        #  to create a dfuse dir and run vol tests from there.
+        # create dfuse container
+        self.start_dfuse(self.hostlist_clients, self.pool, self.container)
+
+        # for test_param in test_list:
+        # testname = test_param[0][1]
+        # client_processes = test_param[1][1]
```
Comment (shimizukko) [inline src/tests/ftest/util/vol_test_base.py:66]: Do we need these lines?

**Comment by mjean308** [comment]
This had previously pass CI and had two approvals.   After a rebase the test failed in SLES 15 func

File: src/tests/ftest/util/vol_test_base.py (hunk 1 above)
Comment (mjean308) [reply src/tests/ftest/util/vol_test_base.py:66]: I will remove them...

**Pull request merged**